cmd/harmony.go

```diff
@@ -89,102 +86,86 @@ func main() {
 	metricsReportURL := flag.String("metrics_report_url", "", "If set, reports metrics to this URL.")
 	versionFlag := flag.Bool("version", false, "Output version info")
 	onlyLogTps := flag.Bool("only_log_tps", false, "Only log TPS if true")
-
-	//This IP belongs to jenkins.harmony.one
-	bcIP := flag.String("bc", "127.0.0.1", "IP of the identity chain")
-	bcPort := flag.String("bc_port", "8081", "port of the identity chain")
-	bcAddr := flag.String("bc_addr", "", "MultiAddr of the identity chain")
-
-	//Leader needs to have a minimal number of peers to start consensus
+	// Leader needs to have a minimal number of peers to start consensus
 	minPeers := flag.Int("min_peers", 100, "Minimal number of Peers in shard")
+	// Key file to store the private key
+	keyFile := flag.String("key", "./.hmykey", "the private key file of the bootnode")
+	flag.Var(&utils.BootNodes, "bootnodes", "a list of bootnode multiaddress")
+
+	// The initial shard ID (TODO: leo) this is a workaround for initial blockchain bootstrap
+	shardID := flag.String("shardID", "0", "the shardID. 0 is beacon chain")
+	// This node is a leader node. (TODO: leo) this is also a workaround for initial blockchain bootstrap
+	isLeader := flag.Bool("leader", false, "false means the node is not a shard leader")
 
 	flag.Parse()
```

Comment:
Is this specifically for beaconLeader (during bootstrap)?

Reply:
yes, I plan to use it that way.